spec/fixtures/responses/queries/freak/freak.json

```diff
@@ -0,0 +1,7 @@
+{
+  "data": {
+    "freak": {
+      "description": "lalala"
+    }
```

Comment:
Where is this data coming from? I don't see any factory being called anywhere in the test.

Reply:
I don't know. :))